mime.types

```diff
@@ -61,6 +61,12 @@ types {
     video/x-ms-wmv                        wmv;
     video/x-msvideo                       avi;
     video/x-matroska                      mkv mk3d;
+    
+    # https://en.wikipedia.org/wiki/HTTP_Live_Streaming
+    application/vnd.apple.mpegurl         m3u8;
```

Comment:
As covering all mime type will not be possible and relevant, we try to avoid at maximum vendor-ed mime types addition.

Reply:
I think dash and hls is part of web now 
https://caniuse.com/#feat=mpeg-dash
https://caniuse.com/#feat=http-live-streaming
I think is better to correct mime type in nginx.

Reply:
Well, 58% and 2% acceptances mean these types are not part of the web yet...

Reply:
It's, cause section of caniuse, mean important for implementation.
+ https://caniuse.com/#feat=mediasource
project https://github.com/videojs/http-streaming using mediasource for streaming. i think part of essential of media on web

Reply:
Not sure to follow...
I'm not saying these types are unknown or unused, just that **on the web** they are not widely and commonly recognized.